reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -321,278 +310,308 @@ class StaticHtmlReporter : Reporter() {
         return outputFile
     }
 
-    private fun createContent(tabularScanRecord: ReportTableModel) =
-            buildString {
-                if (tabularScanRecord.metadata.isNotEmpty()) {
-                    append("<h2>Metadata</h2>")
-                    append("<table class=\"ort-report-metadata\"><tbody>")
-                    tabularScanRecord.metadata.forEach { (key, value) ->
-                        append("""
-                        <tr>
-                            <td>$key</td>
-                            <td>${if (value.isValidUrl()) "<a href=\"$value\">$value</a>" else value}</td>
-                        </tr>
-                        """.trimIndent())
+    private fun renderHtml(reportTableModel: ReportTableModel): String {
+        val document = DocumentBuilderFactory.newInstance().newDocumentBuilder().newDocument()
+
+        document.append.html {
+            lang = "en"
+
+            head {
+                meta(charset = "UTF-8")
+                meta(name = "viewport", content = "width=device-width, initial-scale=1.0")
+                title("Scan Report")
+                style {
+                    unsafe {
+                        raw(css)
```

Comment:
BTW, this seems to remove all indentation from the CSS code, but I'm fine with that.

Reply:
Indentation is acutally removed by `trimIndent` above, but without it was inconvenient editing the result file in IntelliJ because it removes the spaces from lines without content. Removed `raw` anyway, because `+` is shorter and more consistent with the other code.